src/NUnitFramework/framework/Internal/OSPlatform.cs

```diff
@@ -291,7 +291,17 @@ public bool IsWinCE
             get { return platform == PlatformID.WinCE; }
         }
 
-#if !NETCF
+#if NETCF
+        /// <summary>
+        /// Return true if the platform is Xbox
+        /// </summary>
+        public bool IsXbox { get { return false; } }
```

Comment:
Unix and Xbox were added in .NET 3.5,

Do they both need to be conditioned for builds before 3.5?